src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -165,24 +164,24 @@ public void setup() throws Exception {
 
   @After
   public void teardown() throws Exception {
+    logger.info(
```

Comment:
Here's a block of logs for a new run of this teardown method where snapshot IDs, a dataset, and a billing profile all need to be deleted.  This case previously would fail on the first snapshot ID deleted, and no subsequent resource cleanup would run.

```
  2023-02-14 15:47:05,745 INFO  [Test worker] b.t.integration.AzureIntegrationTest: Teardown: trying to delete snapshots [bb071cba-f7f8-437d-8eda-6bc5e38c35e1, 7e768185-eff9-4153-ac95-8755774307a9, 0cb3cdf3-5b11-418e-9ea4-5638515362f6, ca4df033-b829-4979-ba3d-c8aa359eb26c], dataset 11eec9dc-bca3-481c-8b48-894c839e9254, billing profile 73569864-728b-4ed8-8a4a-74f66b37d46e
    2023-02-14 15:47:05,746 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=PUT path=/api/repository/v1/configs/reset user=voldemort body={}
    2023-02-14 15:47:05,905 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=DELETE path=/api/repository/v1/snapshots/bb071cba-f7f8-437d-8eda-6bc5e38c35e1 user=voldemort body=null
    2023-02-14 15:47:06,040 INFO  [Test worker] bio.terra.integration.DataRepoClient: started job: 9onQOQaJSLWHehao2ZFG7g
    2023-02-14 15:47:07,044 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/9onQOQaJSLWHehao2ZFG7g user=voldemort body=null
    2023-02-14 15:47:09,176 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/9onQOQaJSLWHehao2ZFG7g user=voldemort body=null
    2023-02-14 15:47:13,258 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/9onQOQaJSLWHehao2ZFG7g user=voldemort body=null
    2023-02-14 15:47:13,343 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/9onQOQaJSLWHehao2ZFG7g/result user=voldemort body=null
    2023-02-14 15:47:13,423 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=DELETE path=/api/repository/v1/snapshots/7e768185-eff9-4153-ac95-8755774307a9 user=voldemort body=null
    2023-02-14 15:47:13,561 INFO  [Test worker] bio.terra.integration.DataRepoClient: started job: WKFR4VYVQu2ZRCRm_Dq_YQ
    2023-02-14 15:47:14,566 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/WKFR4VYVQu2ZRCRm_Dq_YQ user=voldemort body=null
    2023-02-14 15:47:16,664 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/WKFR4VYVQu2ZRCRm_Dq_YQ user=voldemort body=null
    2023-02-14 15:47:20,770 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/WKFR4VYVQu2ZRCRm_Dq_YQ user=voldemort body=null
    2023-02-14 15:47:20,855 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/WKFR4VYVQu2ZRCRm_Dq_YQ/result user=voldemort body=null
    2023-02-14 15:47:20,954 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=DELETE path=/api/repository/v1/snapshots/0cb3cdf3-5b11-418e-9ea4-5638515362f6 user=voldemort body=null
    2023-02-14 15:47:21,143 INFO  [Test worker] bio.terra.integration.DataRepoClient: started job: XgTq1mzgS-mUMvtGhsfgCw
    2023-02-14 15:47:22,144 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/XgTq1mzgS-mUMvtGhsfgCw user=voldemort body=null
    2023-02-14 15:47:24,226 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/XgTq1mzgS-mUMvtGhsfgCw user=voldemort body=null
    2023-02-14 15:47:28,302 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/XgTq1mzgS-mUMvtGhsfgCw user=voldemort body=null
    2023-02-14 15:47:28,389 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/XgTq1mzgS-mUMvtGhsfgCw/result user=voldemort body=null
    2023-02-14 15:47:28,468 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=DELETE path=/api/repository/v1/snapshots/ca4df033-b829-4979-ba3d-c8aa359eb26c user=voldemort body=null
    2023-02-14 15:47:28,613 INFO  [Test worker] bio.terra.integration.DataRepoClient: started job: Zf7okFmVTI2TPFLfvugrXQ
    2023-02-14 15:47:29,617 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/Zf7okFmVTI2TPFLfvugrXQ user=voldemort body=null
    2023-02-14 15:47:31,693 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/Zf7okFmVTI2TPFLfvugrXQ user=voldemort body=null
    2023-02-14 15:47:35,782 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/Zf7okFmVTI2TPFLfvugrXQ user=voldemort body=null
    2023-02-14 15:47:35,893 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/Zf7okFmVTI2TPFLfvugrXQ/result user=voldemort body=null
    2023-02-14 15:47:35,984 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=DELETE path=/api/repository/v1/datasets/11eec9dc-bca3-481c-8b48-894c839e9254 user=voldemort body=null
    2023-02-14 15:47:36,254 INFO  [Test worker] bio.terra.integration.DataRepoClient: started job: -AjlbDatRkuyM6uaxMHYXA
    2023-02-14 15:47:37,261 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/-AjlbDatRkuyM6uaxMHYXA user=voldemort body=null
    2023-02-14 15:47:39,341 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/-AjlbDatRkuyM6uaxMHYXA user=voldemort body=null
    2023-02-14 15:47:43,423 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/-AjlbDatRkuyM6uaxMHYXA user=voldemort body=null
    2023-02-14 15:47:43,504 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/-AjlbDatRkuyM6uaxMHYXA/result user=voldemort body=null
    2023-02-14 15:47:43,588 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=DELETE path=/api/resources/v1/profiles/73569864-728b-4ed8-8a4a-74f66b37d46e user=voldemort body=null
    2023-02-14 15:47:43,717 INFO  [Test worker] bio.terra.integration.DataRepoClient: started job: Ki1TTlylRhiKzk7upsaStA
    2023-02-14 15:47:44,722 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/Ki1TTlylRhiKzk7upsaStA user=voldemort body=null
    2023-02-14 15:47:44,804 INFO  [Test worker] bio.terra.integration.DataRepoClient: api request: method=GET path=/api/repository/v1/jobs/Ki1TTlylRhiKzk7upsaStA/result user=voldemort body=null

```